templates/supertest/post/post.handlebars

```diff
@@ -27,11 +27,15 @@
       .set('{{headerApiKey.type}}', process.env.{{headerApiKey.name}})
       {{/if}}
       {{#is contentType 'application/json'}}
+      {{#if inputs.body}}
+      .send({{printJSON inputs.body.data}})
```

Comment:
the printJSON helper does not work properly with more complex structures. I would suggest using JSON.stringify from the code.